testsuite/openshift/objects/auth_config.py

```diff
@@ -102,6 +102,39 @@ def add_api_key_identity(self, name, match_label=None, match_expression: MatchEx
             }
         })
 
+    @modify
+    def add_role_rule(self, name: str, role: str, path="^/get", metrics=False, priority=0):
```

Comment:
Please add this method also to the Authorization interface

Reply:
addressed